tests/unittest/others/test_perf_fixed_sequence_length.py

```
@@ -0,0 +1,149 @@
+    )
+
+    config = runner._get_model_yaml_config()
+
+    assert config["kv_cache_config"]["avg_seq_len"] == 1024
+
+
+def test_model_yaml_skips_avg_seq_len_for_older_schema(monkeypatch):
+    from tensorrt_llm.llmapi import llm_args
+
+    class LegacyKvCacheConfig:
+        model_fields = {}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)